frc_characterization/newproject/__init__.py

```diff
@@ -29,6 +31,28 @@
 log_format = "%(asctime)s:%(msecs)03d %(levelname)-8s: %(name)-20s: %(message)s"
 
 
+class Tests(Enum):
+    ARM = "Arm"
+    ELEVATOR = "Elevator"
+    DRIVETRAIN = "Drivetrain"
+    SIMPLE = "Simple"
```

Comment:
Should really be `SIMPLE_MOTOR`

Reply:
[Done ](https://github.com/Piphi5/frc-characterization/blob/Refactor/frc_characterization/newproject/__init__.py#L39)

Reply:
The description should also be "Simple Motor", because that explains what it is better.